src/models/projectModel.js

```diff
@@ -21,6 +21,7 @@ const projectSchema = new mongoose.Schema({
     type: Date,
     required: [true, "A project must have a start date"],
   },
+  clientName: { type: String, required: true },
```

Comment:
Is it possible to move this key somewhere else, because all startdates and enddates are together. And this key is coming between them.